lib/metanorma/plugin/lutaml/liquid_templates/_packages.liquid

```diff
@@ -0,0 +1,150 @@
+= preface
+{{ additional_context.before }}
+
+
+{% for package in context.packages %}
+{% assign before_package_key = 'before;' | append: package.name %}
+{% if additional_context[before_package_key] %}
+{{ additional_context[before_package_key] }}
+{% endif %}
+
+{% assign package_name = package.name | downcase | replace: ":", "" | replace: " ", "_" %}
+{% capture equalsigns %}{% for count in (1..depth) %}={% endfor %}{% endcapture %}{{equalsigns}} {{ package.name }}
+[[rc_{{ package_name }}-model_section]]
+{{equalsigns}}= {{ package.name }}
+
+{% capture requirement_filename %}requirements/requirements_class_{{ package_name }}{% endcapture %}
+{% capture requirement_content %}{% include requirement_filename %}{% endcapture %}
+{% unless requirement_content contains "Liquid error" %}
+{{ requirement_content }}
+{% endunless %}
+
+{% for diagram in package.diagrams %}
+{% capture codelists_image_name %}{{ package.name }} - Code lists{% endcapture %}
+{% capture ade_types_image_name %}{{ package.name }} - ADE Data types{% endcapture %}
+{% if diagram.name == package.name %}
+The UML diagram of the {{ package.name }} module is depicted in
+<<{{ package_name }}-uml>>.
+A detailed discussion of Requirements Class {{ package.name }}
+can be found in the
+link:http://docs.opengeospatial.org/DRAFTS/20-066.html#ug-model-{{ package_name }}-section[CityGML 3.0 Users Guide].
+
+[[{{ package_name }}-uml]]
+.UML diagram of the {{ package.name }} Model.
+
+image::figures/{{ package.name }}.png[align="center"]
+
+{% elsif diagram.name contains ade_types_image_name %}
+The ADE data types provided for the {{ package.name }} module are
+illustrated in <<{{ package_name }}-uml-ade-types>>.
+
+[[{{ package_name }}-uml-ade-types]]
+.ADE classes of the CityGML {{ package.name }} module.
```

Comment:
"CityGML" is not generic, it's the name of a package.

Reply:
Removed diagram blocks with it